oracle_server/oracle_server.go

```diff
@@ -691,18 +687,17 @@ func (os *OracleServer) tryLoadingNewPlugin(f fs.FileInfo, plugConf types.Plugin
 			return
 		}
 
+		os.logger.Info("** Replacing legacy plugin with new one: ", f.Name(), f.Mode().String())
 		// stop the legacy plugin
 		plugin.Close()
 		delete(os.pluginSet, f.Name())
 
-		os.logger.Info("*** Replacing legacy plugin with new one: ", f.Name(), f.Mode().String())
 		pluginWrapper := pWrapper.NewPluginWrapper(os.loggingLevel, f.Name(), os.pluginDIR, os)
 		if err := pluginWrapper.Initialize(); err != nil {
-			os.logger.Error("** Cannot initialize plugin", "name", f.Name(), "error", err.Error())
+			os.logger.Error("Cannot initialize plugin", "name", f.Name(), "error", err.Error())
```

Comment:
I think the first Capital letter is not consistent with other logs and also not the preferred logging style in most golang projects. logging is usually the lowercase. Consistent casing is useful while filtering logs

Reply:
> I think the first Capital letter is not consistent with other logs and also not the preferred logging style in most golang projects. logging is usually the lowercase. Consistent casing is useful while filtering logs

Resolved in bd07764ac76b51a6cd8ae44a169d0b74eb116d63